boards/arm/sama5/giant-board/configs/sdmmc-net-nsh/defconfig

```diff
@@ -0,0 +1,151 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+CONFIG_ARCH="arm"
```

Comment:
This configuration should be removed since it depends on SDMMC which is not available.

Please, save us a lot of problems in the future:  Do not commit any defconfig files that you have not tested.